dissect/target/plugins/apps/container/docker.py

```diff
@@ -35,11 +49,62 @@
     ],
 )
 
-DOCKER_NS_REGEX = re.compile(r"\.(?P<nanoseconds>\d{7,})(?P<postfix>Z|\+\d{2}:\d{2})")
 
+DockerLogRecord = TargetRecordDescriptor(
+    "apps/containers/docker/log",
+    [
+        ("datetime", "ts"),
+        ("string", "container"),
+        ("string", "stream"),
+        ("string", "message"),
+    ],
+)
+
+
+def find_installs(target: Target) -> Iterator[Path]:
+    """Attempt to find additional configured and existing Docker daemon data-root folders.
 
-class DockerPlugin(Plugin):
+    References:
+        - https://docs.docker.com/config/daemon/
     """
+
+    default_config_paths = [
+        # Linux
+        "/etc/docker/daemon.json",
+        "/var/snap/docker/current/config/daemon.json",
+        # Windows
+        "sysvol/ProgramData/docker/config/daemon.json",
+        # Docker Desktop (macOS/Windows/Linux)
+        "$HOME/.docker/daemon.json",
+    ]
+
+    if (default_root := target.fs.path("/var/lib/docker")).exists():
+        yield default_root
+
+    for config_path in default_config_paths:
+        config_file = target.fs.path(config_path)
+
+        if config_path.startswith("$HOME"):
+            for user_details in target.user_details.all_with_home():
+                user_config = user_details.home_path.joinpath(config_path.replace("$HOME/", ""))
+                if user_config.exists():
+                    default_config_paths.append(str(user_config))
+
+        elif config_file.exists():
+            try:
+                config = json.loads(config_file.open("rt").read())
+            except json.JSONDecodeError as e:
+                log.warning(f"Could not read JSON file '{config_file}'")
```

Comment:
```suggestion
                target.log.warning("Could not read JSON file %s", config_file)
```
Logging from the target log object ties it to the target context.